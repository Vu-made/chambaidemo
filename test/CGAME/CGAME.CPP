#include <bits/stdc++.h>
#define N 100005
using namespace std;
int a[N];
int s1 = 0, s2 = 0, n;
int main()
{
    freopen("CGAME.INP","r",stdin);
    freopen("CGAME.OUT","w",stdout);
    cin >> n;
    for (int i=1; i<=n; i++) cin >> a[i];
    int i = 1, j = n, d = 0;
    while (i <= j)
    {
        if (d == 0) s1 += max(a[i],a[j]);
        else s2 += max(a[i],a[j]);

        if (a[i] > a[j]) i++;
        else j--;
        d = 1 - d;
    }
    cout << s1 << " " << s2;
    return 0;
}
