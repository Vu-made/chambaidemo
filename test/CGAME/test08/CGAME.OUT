414152443 415538037